ios/lib/ReactNativeCameraKit/CKCamera.m

```diff
@@ -834,41 +843,56 @@ - (void)didMoveToWindow {
 }
 
 - (void)addSplashScreen {
-    dispatch_async(dispatch_get_main_queue(), ^{
-        self.perfomanceBackground = [[UIView alloc]initWithFrame:self.bounds];
-        self.perfomanceBackground.backgroundColor = [UIColor blackColor];
-        [self addSubview:self.perfomanceBackground];
-        [self bringSubviewToFront:self.perfomanceBackground];
-    });
+    if (self.showFrame) {
+        dispatch_async(dispatch_get_main_queue(), ^{
+            self.perfomanceBackground = [[UIView alloc]initWithFrame:self.bounds];
+            self.perfomanceBackground.backgroundColor = self.perfomanceBackgroundColor;
+            [self addSubview:self.perfomanceBackground];
+            [self bringSubviewToFront:self.perfomanceBackground];
+        });
+    }
 }
 
 - (void)setScannerOptions:(NSDictionary *)scannerOptions {
-    if (scannerOptions[offsetForScannerFrame]) {
-        self.frameOffset = [scannerOptions[offsetForScannerFrame] floatValue];
+    if (scannerOptions[sFrameWidth]) {
+        self.frameWidth = [scannerOptions[sFrameWidth] floatValue];
+    }
+    if (scannerOptions[sFrameHeight]) {
+        self.frameHeight = [scannerOptions[sFrameHeight] floatValue];
+    }
+    if (scannerOptions[sFrameLeft]) {
+        self.frameLeft = [scannerOptions[sFrameLeft] floatValue];
     }
-    if (scannerOptions[heightForScannerFrame]) {
-        self.heightFrame = [scannerOptions[heightForScannerFrame] floatValue];
+    if (scannerOptions[sFrameTop]) {
```

Comment:
Same here, what's the meaning of `sFrameTop`? starting frame? scanner frame? and what is top? top frame? top edge?

Reply:
this is string constants instead of string values. All constants are defined